src/components/Home/Header.module.css

```diff
@@ -97,6 +97,9 @@
     margin-top: 10%;
   }
   .title {
-    font-size: 3.5rem;
+    font-size: calc(var(--font-size-h1) * 0.666666);
```

Comment:
just checking, why are we multiplying a var for font sizes? should we not just set a size?

Reply:
trying to keep the proportion between Maria's design and the existing code, but since the code is inside a @media block it is possible to set a fixed value

Reply:
just set the value that Maria used in the design

Reply:
I can use a fixed font but it needs to be smaller than the size used in the design. The text is too long and it will require major changes in the UI to fit it in, specially the one below the partners.

Reply:
Set the fonts to a fixed size

Reply:
Set fonts to rem